selftests/pre_release/jobs/yaml/check_content_features.yaml

```diff
@@ -0,0 +1,11 @@
+features: !mux
+  tap:
+    namespace: 'job.run.result.tap.include_logs'
+    file: 'results.tap'
+    content: "Command '/bin/true' finished with 0"
+
+status: !mux
+  enabled:
+    value: 'on'
+  disabled:
+    value: 'off'
```

Comment:
This syntax is a little odd to me, but probably because I'm not familiar with this mux format. Maybe after reading some documentation and examples will be clear.

After we decide on the format and syntax, IMO nice documentation explaining how this works and what are the available options would be nice. (besides the mux documentation)

Reply:
I agree, there is no way to fully understand the tests if the tester do not understand the variants.